.github/workflows/ci.yml

```diff
@@ -37,7 +37,8 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: wyvox/action-setup-pnpm@v3
-      - run: pnpm --filter '*' test 
+      - run: pnpm jest
```

Comment:
Can we use vitest instead of jest?

Jest has been a royal pain, and i don't want to give them more usage:/

Reply:
sure, I like vite